rosidl_generator_rs/resource/Cargo.toml.em

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "@(package_name)"
+version = "0.2.0"
```

Comment:
Can you add an EmPy comment with a reference to https://github.com/ros2-rust/ros2_rust/issues/116 ?

Reply:
Good idea, done.

Reply:
Thanks.